Vagrantfile

```diff
@@ -2,16 +2,21 @@ VAGRANTFILE_API_VERSION = "2"
 
 Vagrant.configure("2") do |config|
   config.vm.box = "ubuntu/trusty64"
-  config.vm.provision :shell, path: "scripts/start-services.sh"
   config.vm.network :forwarded_port, host: 3500, guest: 3500 # Thimble
   config.vm.network :forwarded_port, host: 5432, guest: 5432 # Postgres
   config.vm.network :forwarded_port, host: 2015, guest: 2015 # publish.webmaker.org
   config.vm.network :forwarded_port, host: 8001, guest: 8001 # Published projects
   config.vm.network :forwarded_port, host: 1234, guest: 1234 # id.webmaker.org
-  config.vm.network :forwarded_port, host: 3000, guest: 3000 #login.webmaker.org
+  config.vm.network :forwarded_port, host: 3000, guest: 3000 # login.webmaker.org
   config.vm.provider "virtualbox" do |v|
-    v.memory = 2048
-    v.cpus = 2
+    v.memory = 1536
```

Comment:
This value is intriguing. Should we mention in the readme that people will need 1.5GB of free RAM? For most machines, not a problem but it might be an issue for low-RAM laptops.